pub-critic/src/app/components/LikeAndDislikeRow/LikeAndDislike.tsx

```diff
@@ -111,29 +107,16 @@ export const LikeAndDislike = ({
     setLoading(false);
```

Comment:
This can be simplified a bit

Reply:
will change this in likes and dislikes pr